src/lib/wallet/GoodWalletClass.js

```diff
@@ -249,73 +252,50 @@ export class GoodWallet {
 
         log.info('GoodWallet setting up contracts:')
 
+        const makeContract = (abi, name, defaultAddress = null) => {
```

Comment:
those things are repeated each time we add contract. we have 6-7 of them in total
+ now we have to check does address exists in the current network (it may be absent on mainnet/goerli), 

so I decided to create an helper fns